docs/error/404.md

```diff
@@ -0,0 +1,3 @@
+# Page Removed or Relocated or Never Existed
+
+If you're here that means you requested a page that doesn't exist. Sorry about that! Maybe try the search box to find what you're looking for.
```

Comment:
Suggest to also give a link to the master website: "Try the search box to find what you're looking for, or navigate to [mxnet home page](mxnet.io)"